docs/source/x_ext.rst

```diff
@@ -256,6 +256,25 @@ The ``id`` values for in-flight offloaded instructions are required to be unique
 The ``id`` values are required to be incremental from one issue transaction to the next.
 The increment may be greater than one.
 If the next ``id`` would be greater than the maximum value (``2**X_ID_WIDTH - 1``), the value of ``id`` wraps.
+A new ``id`` value is not allowed to be greater than the oldest in-flight instruction, if a wrap has occurred since the oldest in-flight instruction was issued.
+If the oldest in-flight instruction is :math:`id_o`, and the newest is ``id_n``, then the next instruction with ``id_n+1`` must satisfy the following conditions:
+
+.. math::
+  \begin{gather}
+    id_{n+1} > id_{n} \text{ or } id_{n+1} < id_{o}, \text{ if } id_{n} > id_{o}\\
+    id_{n+1} > id_{n} \text{ and } id_{n+1} < id_{o}, \text{ if } id_{n} < id_{o}
+  \end{gather}
+
+The first condition applying to cases where the ``id`` has wrapped since the oldest in-flight instruction was issued, and the second where no wrap occurred.
```

Comment:
In 268 I think the two cases might be reversed. My suggestion would be smth. like:

> The first condition applying to cases where the ``id`` has not wrapped since the oldest in-flight instruction was issued, and the second where a wrap occurred.

Other than that, I think all problems with the previous ambiguity are now clarified nicely.

Reply:
thanks for the comment. this is what we really need to make sure is correct as it is quite confusing.

however, i think it is correct as it is. id_{n} < id_{o} can only happen if the id value wrapped.
Please double check again and let me know if you agree